SDK/AppCenterData/Microsoft.AppCenter.Data.Shared/Document.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.AppCenter.Data
```

Comment:
https://msmobilecenter.visualstudio.com/Mobile-Center/_wiki/wikis/Mobile-Center.wiki?wikiVersion=GBwikiMaster&pagePath=%2FHome%2FTeams%2FMBaaS%2FData%2FSpecifications%2FJava%20SDK%20Signatures&pageId=1925

Some spec changes are underway, c# should reflect that, will add more specific comments on this file.